File: grpc-server/src/definition/arcade.ffa.entity.rs
#[derive(serde::Serialize, serde::Deserialize)]
#[serde(rename_all = "snake_case")]
#[derive(Debug, Clone, PartialEq, Eq)]
pub struct FfaPlayerStats {
    pub uuid: ::uuid::Uuid,
    pub kills: i32,
    pub deaths: i32,
    pub assists: i32,
    pub killstreak: i32,
    pub longest_killstreak: i32,
}

File: kotlin-grpc-client/src/main/kotlin/org/hexalite/network/definition/arcade.ffa.entity.kt
@file:kotlinx.serialization.UseSerializers(org.hexalite.network.common.serialization.UUIDSerializer::class, org.hexalite.network.common.serialization.ChronoInstantSerializer::class)
package org.hexalite.network.definition.arcade.ffa.entity
@kotlinx.serialization.Serializable
data class FfaPlayerStats (
    @OptIn(kotlinx.serialization.ExperimentalSerializationApi::class)
    @kotlinx.serialization.cbor.ByteString
    val uuid: java.util.UUID,
    val kills: Int,
    val deaths: Int,
    val assists: Int,
    val killstreak: Int,
    @kotlinx.serialization.SerialName("longest_killstreak")
    val longestKillstreak: Int,
)
